src/helpers/update-pull-requests.ts

```diff
@@ -11,37 +18,26 @@ type IssueEvent = {
   commented_at?: string;
 };
 
-async function getPullRequestDetails(context: Context, { repo, owner, issue_number: pullNumber }: IssueParams) {
-  return (
-    await context.octokit.pulls.get({
-      repo,
-      owner,
-      pull_number: pullNumber,
-    })
-  ).data;
-}
-
 export async function updatePullRequests(context: Context) {
-  const pullRequests = await context.adapters.sqlite.pullRequest.getAll();
+  if (!context.config.repos.monitor.length) {
+    return context.logger.info("No organizations or repo have been specified, skipping.");
+  }
+
+  const pullRequests = await getOpenPullRequests(context, context.config.repos);
 
   if (!pullRequests?.length) {
     return context.logger.info("Nothing to do.");
   }
-  for (const pullRequest of pullRequests) {
+  for (const { html_url } of pullRequests) {
     try {
```

Comment:
Could this be made more readable It's sort of hard to grasp immediately what's going on. We used to get warnings for these types of deep nested ifs and complexity

Reply:
For each `pullRequests` item I just pick up the destructured value of the item. Is that not readable, would you prefer me to use `pullRequest.html_url`?

Reply:
Sorry terrible aiming, I meant all of the nested ifs within this try block

Reply:
Yes I agree I will split that. Interesting that `sonar` didn't complain because it is enabled.